modules/local/predict_epitopes.nf

```diff
@@ -0,0 +1,77 @@
+process PREDICT_EPITOPES {
+    label 'process_low'
+
+    // TODO: conda
+    conda (params.enable_conda ? { exit 1 "Conda is currently not available for metapep" } : null)
+    container 'skrakau/metapep:dev'
+    
+    input:
+    path(peptides)
+
+    output:
+    path "*predictions.tsv",            emit:   ch_epitope_predictions
+    path "*prediction_warnings.log",    emit:   ch_epitope_prediction_warnings
+    path "versions.yml",                emit:   versions
+
+    script:
+    def pred_method           = params.pred_method
+    switch (pred_method) {
+        case "syfpeithi":
+            pred_method_version = "1.0";
+            break;
+        case "mhcflurry":
+            pred_method_version = "1.4.3";
+            break;
+        case "mhcnuggets-class-1":
+            pred_method_version = "2.3.2";
+            break;
+        case "mhcnuggets-class-2":
+            pred_method_version = "2.3.2";
+            break;
+        default:
+            exit 1, "Epitope prediction method specified with --pred_method not recognized."
```

Comment:
while that is fine to exit here, it would be great to make sure those are checked earlier, specifically on pipeline startup, using e.g. controlled vocabluary in `nextflow_schema.json`. (maybe its already in, did not check!)